Documentation/contributing/release/backports.rst

```diff
@@ -103,9 +104,9 @@ One-time Setup
    +--------------------------------------------------------------+-----------+---------------------------------------------------------+
    | python3                                                      | No        | `Python Downloads <https://www.python.org/downloads/>`_ |
    +--------------------------------------------------------------+-----------+---------------------------------------------------------+
-   | `PyGithub <https://pypi.org/project/PyGithub/>`_             | No        | ``pip3 install PyGithub``                               |
+   | `PyGithub <https://pypi.org/project/PyGithub/>`_             | Yes       | ``pip3 install PyGithub``                               |
    +--------------------------------------------------------------+-----------+---------------------------------------------------------+
-   | `Github hub CLI (>= 2.8.3) <https://github.com/github/hub>`_ | No        | N/A (OS-specific)                                       |
+   | `Github hub CLI (>= 2.8.3) <https://github.com/github/hub>`_ | Yes       | N/A (OS-specific)                                       |
```

Comment:
I'm not sure that `hub` is required. At least I haven't had to use it since I created the PR by hand. `PyGithub` is for sure required for marking PRs else it's really tedious

Reply:
If you grep for `hub`, it's definitely used. It opens the backport PRs.

Reply:
Yes, hub is used.

Reply:
I don't think my initial comment was read properly. 

Yes, it's used if you run `contrib/backporting/submit-backport`. However that itself is not required since the docs branch into:
- use submit-backport (hub required)
- open PR manually (and copy over the contents of the file)

If we mark hub as required (it's annoying to install on ubuntu after snap install deprecation) we should remove the "create PR manually" section, or not mark hub as required.

Reply:
As an additional data point, I opened the backport PR just fine today without hub purely following the docs.

Reply:
Honestly, we should do away with the "create manually" section. I've only seen it causes problems for people. The process is automated in the scripts; might as well leverage it

Reply:
Another idea would be to convert `hub` to a dockerized app? We'd have to make sure that the docker app can access the user's git config

Reply:
I'm fully supportive of leaning into hub and removing the manual section. I was just trying to highlight the issue so we resolve it.

Reply:
> Honestly, we should do away with the "create manually" section. I've only seen it causes problems for people. The process is automated in the scripts, might as well leverage it

Fwiw, I never used hub, always created PRs manually and I don't recall having issues.

Reply:
I'd also be in favor of removing the "manual" section. There are regularly backport PRs opened against master, sending tons of notifications to everyone in the process, and without the proper label updates.

If we containerize `hub`, we might as well include all the backporting scripts.

Reply:
> If we containerize hub, we might as well include all the backporting scripts.

That is even better.

Reply:
> Fwiw, I never used hub, always created PRs manually and I don't recall having issues.

@rolinh the most common issue is accidentally creating the backport PR against the wrong branch (ie opening the PR against master instead of the target branch). Beyond that, there's also making sure the labels are set correctly for the backported PRs, and making sure that the PR description contains the summary of the backported PRs and importantly, the section with the command to update the backports to "done" once the backport PR is merged, which is not only for label management but is also used to correlate the backport + original PRs in release notes. The script handles all of these bits and pieces for you so you don't need to make sure you line them up correctly.

Reply:
> ...opening the PR against master instead of the target branch

It happen to me several times. Fortunately github allows changing branches after PR was open, however reviewer requests end-up being messed up.

Reply:
I've been using a container every time, and recently published the images here:
 https://hub.docker.com/repository/docker/errordeveloper/cilium-backport-tools

I'd be happy to turn that one-off image into something we can all use, as it sounds this would be of interest to people.